metasyn/distribution/base.py

```diff
@@ -29,7 +29,7 @@ class BaseDistribution(ABC):
     """Abstract base class to define a distribution.
 
     All distributions should be derived from this class, and should implement the following methods:
-    _fit, draw, _param_dict, _param_schema, default_distribution and __init__.
+    `_fit`, `draw`, `_param_dict`, `_param_schema`, `default_distribution` and `__init__`.
```

Comment:
Maybe this is useful for sphinx crosslinking?

https://stackoverflow.com/questions/21289806/link-to-class-method-in-python-docstring